jnigen/java/src/main/java/com/github/dart_lang/jnigen/apisummarizer/util/SearchUtil.java

```diff
@@ -29,7 +28,6 @@ public static Optional<List<File>> findFilesInPath(
     return Optional.empty();
   }
 
-  /// find files in jar
   public static Optional<List<ZipEntry>> findFilesInJar(
       String qualifiedName, JarFile jar, String suffix) {
     String relativePath = qualifiedName.replace(".", "/");
```

Comment:
also here

Reply:
It seems Jar entries use / in separators regardless of the OS. Probably a backslash will work but having parity with getName() etc methods on JarEntry is a good idea.

```
jshell> var j = new JarFile(".dart_tool/jnigen/ApiSummarizer.jar")
j ==> java.util.jar.JarFile@146ba0ac

jshell> j.stream().limit(5).forEach(e -> System.out.println(e))
META-INF/
META-INF/MANIFEST.MF
com/
com/github/
com/github/dart_lang/
```

Reply:
It seems Jar entries use / in separators regardless of the OS. Probably a backslash will work but having parity with getName() etc methods on JarEntry is a good idea.

```
jshell> var j = new JarFile(".dart_tool/jnigen/ApiSummarizer.jar")
j ==> java.util.jar.JarFile@146ba0ac

jshell> j.stream().limit(5).forEach(e -> System.out.println(e))
META-INF/
META-INF/MANIFEST.MF
com/
com/github/
com/github/dart_lang/
```